lib/internal/streams/writable.js

```diff
@@ -638,7 +644,7 @@ function onwrite(stream, er) {
 }
 
 function afterWriteTick({ stream, state, count, cb }) {
-  state.afterWriteTickInfo = null;
```

Comment:
We should probably still set it to null (the contents of the kAfterWriteTickInfoValue symbol) to prevent memory leak potential?